### DISCUSSION TITLE
REQUESTED PERMISSIONS

### DISCUSSION THREAD

**Discussion by yangkai69** [WalletConnect/walletconnect-monorepo#3689] [Developer Support]
Hello, I ran the wallet demo and scanned the QR code, displaying requested permissions, but I have already given all permissions in the settings. What should I do now? Does this have anything to do with my presence in China? I am using VPN

**Comment by glitch-txs** [comment]
hi could you provide more information fo the following:
- which wallet demo where you using?
- Which application where you connecting to
- Do you have any video/screenshot of the error?

**Reply by yangkai69** [reply]
I am using the Android demo on the official website and connecting to Dapp(Uniswap).
Take a screenshot of the app and see the next comment。

**Comment by yangkai69** [comment]
I am using the Android demo on the official website and connecting to Dapp(Uniswap). The screenshot is
![WechatIMG209](https://github.com/WalletConnect/walletconnect-monorepo/assets/16103086/85bc7772-42e0-4385-97c6-fc837eb42825)

**Comment by yangkai69** [comment]
I have solved this problem. It's a UI issue. Just change the UI for the demo

**Comment by LisaMA95** [comment]
If you use a VPN, can't they steal funds from your wallet?
